src/logger/loggerTaskEx.c

```diff
@@ -246,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
> if they have a config that coincidentally does not include the telemetry sample rate

How is that even possible?  Where would a user set the telemetry sample rate exactly?  I certainly don't see such a location in the App.  One plausible cause is if _all_ channel sample rates are disabled, but even then you would have the interval and UTC time channel (which can not be disabled) so that is really not a valid scenario.

I remember this _was_ an issue long ago, but I'm not sure if it is one now.  Thoughts?